docs/src/main/asciidoc/spring-di.adoc

```diff
@@ -1,28 +1,27 @@
-////
-This guide is maintained in the main Quarkus repository
-and pull requests should be submitted there:
-https://github.com/quarkusio/quarkus/tree/master/docs/src/main/asciidoc
-////
-= Quarkus - Quarkus Extension for Spring DI API
+//TODO: need to use product attributes, but also keep compatibility with community ones
 
-include::./attributes.adoc[]
-:extension-status: preview
+// comment out to show community version
+:product-docs:
+:doctype: book
 
-While users are encouraged to use CDI annotations for injection, Quarkus provides a compatibility layer for Spring dependency injection in the form of the `spring-di` extension.
+//Top-Level Assembly
+[id="using-the-quarkus-extension-for-spring-di-api_{context}"]
+= Using the Quarkus Extension for Spring DI API
 
-This guide explains how a Quarkus application can leverage the well known Dependency Injection annotations included in the Spring Framework.
+ifdef::context[:parent-context: {context}]
 
-include::./status-include.adoc[]
+:context: using-the-quarkus-extension-for-spring-di
 
-== Prerequisites
+include::./attributes.adoc[]
 
-To complete this guide, you need:
+[role="_abstract"]
+The `quarkus-spring-di` extension provides a compatibility layer for Spring Dependency Injection, that you can use as an alternative to MicroProfile CDI annotations in your Quarkus application.
```

Comment:
Per Eclipse license the first occurrence of 'MicroProfile' must be 'Eclipse MicroProfile'